bundle/language_fr.properties

```diff
@@ -492,6 +492,7 @@ angal.lab.pleaseselectapatient
 angal.lab.positive                                                               = Positif
 angal.lab.printreport                                                            = Estampage rapport
 angal.lab.printtable                                                             = Estampage tableau
+angal.lab.print																	 = Estampage
```

Comment:
convertion space->tabs happening here

Reply:
Fixed with the push c9b6c019656dba1594baf288c36cf9d381f4130d